src/CBT.NuGet/Tasks/ImportBuildPackages.cs

```diff
@@ -120,7 +121,12 @@ private bool GenerateBuildPackageImportFile()
 
             foreach (BuildPackageInfo buildPackageInfo in ModulePaths.Select(BuildPackageInfo.FromModulePath).Where(i => i != null))
             {
-                ProjectPropertyElement enableProperty = propertyGroup.AddProperty(buildPackageInfo.EnablePropertyName, "false");
+                bool defaultEnableValue = false;
```

Comment:
Okay I finally understand what you're trying to do.  If its a module because it has a `module.config`, its imported by default.  However, there's no guarantee that people will be using the `CBT.NuGet` module in their code bases.  So if all modules have a layout like:

```
CBT.Traversal
└───build
            CBT.Traversal.props
            module.config
            Traversal.targets
```

And we eliminate the proprietary `build.props`, then we want them to work with or without the `CBT.NuGet` module.  So CBT.Core should import `CBT.Traversal.props` like it does `build.props` and `CBT.NuGet` should only generate imports for build packages that are not modules.